openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.11/2-eventing-core.yaml

```diff
@@ -3881,6 +3881,29 @@ spec:
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-trusted-cabundle
```

Comment:
this was added _manually_, and will be included differently (e.g. perhaps via midstream, perhaps we patch to this file).

for testing, manual add is pretty much good enough